src/main/scala/com/redis/SetOperations.scala

```diff
@@ -6,6 +6,43 @@ import com.redis.serialization._
 trait SetOperations extends SetApi {
   self: Redis =>
 
+  override def sort[A](key: String,
+                       limit: Option[(Int, Int)] = None,
+                       desc: Boolean = false,
+                       alpha: Boolean = false,
+                       by: Option[String] = None,
+                       get: List[String] = Nil)(implicit format: Format, parse: Parse[A]): Option[List[Option[A]]] = {
+
+    val commands: List[Any] = makeSortArgs(key, limit, desc, alpha, by, get)
+    send("SORT", commands)(asList)
+  }
+
+  private def makeSortArgs(key: String,
+                           limit: Option[(Int, Int)] = None,
+                           desc: Boolean = false,
+                           alpha: Boolean = false,
+                           by: Option[String] = None,
+                           get: List[String] = Nil): List[Any] = {
+    List(List(key), limit.map(l => List("LIMIT", l._1, l._2)).getOrElse(Nil)
+      , (if (desc) List("DESC") else Nil)
+      , (if (alpha) List("ALPHA") else Nil)
+      , by.map(b => List("BY", b)).getOrElse(Nil)
+      , get.flatMap(g => List("GET", g))
+    ).flatten
+  }
+
+  override def sortNStore[A](key: String,
+                             limit: Option[(Int, Int)] = None,
+                             desc: Boolean = false,
+                             alpha: Boolean = false,
+                             by: Option[String] = None,
+                             get: List[String] = Nil,
+                             storeAt: String)(implicit format: Format, parse: Parse[A]): Option[Long] = {
+
+    val commands = makeSortArgs(key, limit, desc, alpha, by, get) ::: List("STORE", storeAt)
+    send("SORT", commands)(asLong)
+  }
+
```

Comment:
Why is `sort` part of `SetOperations` ?

Reply:
Well you are right it should be some other part of api. Which one would you recommend?

Reply:
I think `BaseOperations` will be better ..

Reply:
moved back :)